ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -65,3 +80,17 @@ PMP_S_VSCF_ISV:
   - mixing_valve_percentage_command
   - chilled_supply_water_temperature_setpoint
   - chilled_supply_water_temperature_sensor
+
+PMP_TW_TPKD_1:
+  id: "5496288685028352000"
+  description: "Non-standard type where only the device alarm, mode and status are being operated and monitored."
+  is_canonical: true
```

Comment:
if this is non-standard, it should not be listed as canonical.

Reply:
Description changed.